miio/miot_device.py

```diff
@@ -29,6 +29,18 @@ def get_properties_for_mapping(self) -> list:
         return self.get_properties(
             properties, property_getter="get_properties", max_properties=15
         )
+    
+    def get_property(self, property_key: str) -> dict:
```

Comment:
The return type should be `Dict` (import from typing is necessary) until python 3.9, iirc.